bin/client.py

```diff
@@ -292,10 +292,16 @@ def main():
                           default='/etc/apel/sender.cfg')
 
     opt_parser.add_option('-l', '--log_config',
-                          help='location of logging config file (optional)',
-                          default='/etc/apel/logging.cfg')
+                          help='DEPRECATED - location of logging config file (optional)',
+                          default=None)
 
     options, unused_args = opt_parser.parse_args()
+
+    # Deprecating functionality.
+    old_log_config_default_path = '/etc/apel/logging.cfg'
```

Comment:
This "constant" only get used once in the next line, and the path is longer than the variable name! I'd say just fold that into the `if` statement.